docusaurus.config.js

```diff
@@ -20,6 +20,71 @@ module.exports = {
       },
     },
   },
+  headTags: [
```

Comment:
What's the headTag? is it required by 2.2.0?

Reply:
added support for https://docusaurus.io/docs/api/docusaurus-config#headTags

I removed the plugin